lib/utils/constants_file.dart

```diff
@@ -0,0 +1,6 @@
+abstract class Constants{
+  static const double radius = 100.0;
+  static const double nameFontSize = 45.0;
+  static const double textFontSize = 25.0;
+  static const double emailFontSize = 20.0;
+}
```

Comment:
ctrl + alt + L here to fix the missing line, please

Reply:
Done